client-v2/src/main/java/com/clickhouse/client/api/internal/Gauge.java

```diff
@@ -18,4 +18,9 @@ public void set(long value) {
     public long getLong() {
         return value;
     }
+
+    @Override
+    public String getString() {

```

Comment:
Just want to note: 
this is the first sign that something wrong with `getString()` in interface: we have to adopt existing classes to a new version. If we extending interface in the future we should keep it backward compatible as much as possible.